rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -133,6 +136,9 @@
     private FastBridgeFederationInformation fastBridgeFederationInformationsToSave = null;
     private long receiveHeadersLastTimestamp = 0;
 
+    private Long nextUtxoExpirationCheckpointHeight;
```

Comment:
I suggest using long instead of Long as we should never receive a null value here.

Reply:
Using long Will Require Me To Initialise It To Zero
Then Check If Its > 0 Before Setting Or Saving To Storage.. Is That Okay?

Reply:
It will initialize to `0` by default, then just check value > 0 in the setter. Makes it simpler

Reply:
Resolved